spec/issues/923_params_schema_definition_for_delete_action_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe '#923 Body params for DELETE action' do
+  let(:app) do
+    Class.new(Grape::API) do
+      params do
+        requires :post_id, type: Integer
+        requires :query, type: String, documentation: { type: 'string', param_type: 'body' }
```

Comment:
For the spec, I chose a simple example. However, it can also be a complex object, like:
```json
{
  "filter": {
    "name_include": "foo",
    "created_at_lt": "2000-01-01"
  },
  "reason": "I don't need this anymore",
  "retention": "6 days",
  "keep_assets": true
}
```